app/utils/achievementConstants.ts

```diff
@@ -198,7 +198,7 @@ const AchievementsInfo: Record<string, AchievementData[]> = {
       hidden: false,
     },
     {
-      name: 'Studentenes forkjemper',
+      name: 'Studentenes\nforkjemper',
```

Comment:
I would think this is better to achieve by restricting the size of the box with css instead.